components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -167,7 +167,7 @@ export default function () {
                         </div>
                         <button onClick={showStartWSModal} className="ml-2">New Workspace</button>
                     </div>
-                    <ItemsList className="app-container">
+                    <ItemsList className="app-container pb-40">
```

Comment:
**thought**: Not sure why this didn't work with margin but using padding here does the trick.